### ISSUE TITLE
PowerNukkit

### ISSUE THREAD

**Issue by feanchy** [wode490390/ClassicVillagePopulator#5]
Make a version with new villages for PowerNukkit

**Comment by MonoAdrian23** [comment]
Works great with PowerNukkit 
Thanks for it

**Comment by hinexmine** [comment]
> Отлично работает с PowerNukkit
> Спасибо за это

Yes, works fine with PowerNukkit, I just asked to make the same plugin but with villages from the new version, since new blocks are working on PowerNukkit

**Comment by feanchy** [comment]
> Works great with PowerNukkit
> Thanks for it

Yes, works fine with PowerNukkit, I just asked to make the same plugin but with villages from the new version, since new blocks are working on PowerNukkit

**Comment by MonoAdrian23** [comment]
I dont think this Feature is comming

**Comment by DerCooleVonDem** [comment]
I currently porting this plugin to php and it would be a insane amount of work to create newer villages...

**Comment by MonoAdrian23** [comment]
Fuck PHP 
What the hell xd
